packages/kbn-docs-utils/src/api_docs/mdx/write_deprecations_doc_by_plugin.ts

```diff
@@ -9,17 +9,17 @@
 import moment from 'moment';
 import { ToolingLog } from '@kbn/tooling-log';
 import dedent from 'dedent';
-import fs from 'fs';
+import fs from 'fs/promises';
```

Comment:
Nit: I like using `import Fsp from 'fs/promises'` if for no other reason because there are APIs which are only available via `fs` so there are cases where both are needed, and `Fsp` makes it very clear that this isn't the classic `fs` module that many are very familiar with.